code/modules/awaymissions/mission_code/TheFactory.dm

```diff
@@ -218,34 +218,10 @@
 	hud_type = list(DATA_HUD_SECURITY_ADVANCED, DATA_HUD_DIAGNOSTIC_BASIC, DATA_HUD_MEDICAL_ADVANCED)
 	hud_trait = list(TRAIT_SECURITY_HUD, TRAIT_MEDICAL_HUD)
 
-/obj/item/soundsynth/ancient//the original soundsynth code says it was ported from /vg/station. I guess I should mention it or I'll be sent to Brazil. https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/game/objects/items/devices/sound_synth.dm
-	name = "ancient sound synthesizer"
-	desc = "A device that is able to create sounds. This one is an ancient relic and has no restrictions."
-	shiftpitch = 0
-	sound_list = list()
-
 /obj/item/soundsynth/examine(mob/user)
 	. = ..()
 	. += "<span class='info'>Alt+Click on it to change volume.</span>"
```

Comment:
This description is now obsolete. Better change it to something like this (and move it to a more proper place):

```suggestion
	. += "<span class='info'>Alt+Click on it to pick a sound.</span>"
```

Obviously, you can remove this proc.

Reply:
Wait a minute, why the fuck is this overriding the normal soundsynth.... in factory code